app/static/js/utils/Byteflag.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+var Byteflag = {};
+
+/**
+ * add: Add bit to byteflag.
```

Comment:
This description makes it seem like it's done in-place. Maybe something along the line, "Return new byteflag with newBit added to existing flag"